mockup/patterns/moment/pattern.js

```diff
@@ -146,5 +147,15 @@ define([
   });
 
   return Moment;
-
 });
+
+function lazyLoadMomentLocale() {
```

Comment:
@ale-rt , @agitator: oops, this function leaks to the global object. My goal was to hide some of the imperative code, so all patterns looked more similar between each other. I'll fix it in #890.